src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
+  sizeXS?: SpacingUnit | 0
```

Comment:
Do you think encoding the environment into this API prop is necessary? It might be a little less overhead to think about this component as an extension of `<Box>` and then see if this is still needed to achieve the same end result. Then people only have to think about one system.